internal/agent/device/systeminfo/network.go

```diff
@@ -268,14 +268,22 @@ func getDefaultRouteIPv4(reader fileio.Reader) (*DefaultRoute, error) {
 
 		// is the default route (destination 0.0.0.0)
 		if fields[1] == "00000000" {
+			interfaceName := fields[0]
+			gatewayHex := fields[2]
+
+			// skip loopback interfaces
+			if isLoopback(interfaceName) {
+				continue
```

Comment:
question: should we add some trace logging here, not sure exactly how far away a logger is.

Reply:
It's only one level up, I'll pass it down and add logging.